snuba/web/query.py

```diff
@@ -55,6 +74,7 @@ def raw_query(
     query: DictClickhouseQuery,
     reader: Reader[ClickhouseQuery],
     timer: Timer,
+    query_list: List[ClickhouseQueryMetadata],
```

Comment:
```suggestion
    query_list: MutableSequence[ClickhouseQueryMetadata],
```

Unless you _need_ an actual `List` here, it's better to just specify the interface that you are expecting — in this case this also helps because it explicitly declares that you expect to mutate this parameter (or allow some other part of the stack to mutate it.)